### DISCUSSION TITLE
How to add new language

### DISCUSSION THREAD

**Discussion by desis123** [serp-ai/bark-with-voice-clone#74] [Q&A]
I have seen Bangla is in your coming soon list. But I have not seen any update for a long time. So my question is how new language like Banlga can be trained

**Comment by devinschumacher** [comment]
@desis123 We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉Check it out: https://www.youtube.com/watch?v=84JVGtkapik
